ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
@granders Sure, I will update the patch to specify sudo_user per ssh_command. I have two questions:

- Do you think it needs to be specified in the RemoteAccount() as well?
- If I understand it right, you have a use case for which you create RemoteAccount with user = A and needs to execute ssh_command with user = root for in some cases and user = A in other cases. If A != root, how to you currently address the problem without this patch? If A == root, then this patch can address the use case without having sudo_user per ssh_command, right?